ovirt/provider.go

```diff
@@ -38,6 +38,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("OVIRT_URL", os.Getenv("OVIRT_URL")),
 				Description: "Ovirt server url",
 			},
+			"headers": &schema.Schema{
+				Type:        schema.TypeMap,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OVIRT_HEADERS", os.Getenv("OVIRT_HEADERS")),
```

Comment:
AFAIK, `os.Getenv()` will return a string which could be converted to `TypeMap`. Maybe we could leave  it without a default value or default function. What's your thoughts, please let me know.

Reply:
I had a look to terraform source and decided to provide a `Default` (empty map) instead of a `DefaultFunc`. Seems to be more consistent with the rest 🙂

Reply:
Yes, that does make sense.